eng/versioning/version_client.txt

```diff
@@ -44,6 +44,6 @@ com.azure:perf-test-core;1.0.0-beta.1;1.0.0-beta.1
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: the unrelease dependencies do not require the empty line as their versions
 # will not be manipulated with the automatic PR creation code.
-unreleased_com.azure:azure-core;1.3.0-beta.2
-unreleased_com.azure:azure-core-amqp;1.1.0-beta.1
-unreleased_com.azure:azure-core-http-netty;1.4.0-beta.2
+unreleased_com.azure:azure-core;1.3.0
```

Comment:
After you release core I assume you are planning to make another update to remove these unreleased versions before you try to release the other libraries that currently depend on them?

Reply:
Yes, the unreleased versions will be removed.

Reply:
@weshaggard, @alzimmermsft knows that after the core libraries have been released that these unreleased dependencies (since they're GA and that'll be the new dependency version) will need to be removed. This has to happen before anything with these dependencies is released.